neon-animated-pages.html

```diff
@@ -32,24 +32,28 @@
         position: relative;
       }
 
-      :host > ::content > * {
+      :host > ::content > *,
```

Comment:
you shouldn't need to style `::content` here and below. Polymer 1.7+ will handle the conversion of `::slotted` to `::content` for you 👌